x/bank/alias.go

```diff
@@ -15,29 +16,28 @@ const (
 	ModuleName               = types.ModuleName
 	RouterKey                = types.RouterKey
 	DefaultParamspace        = types.DefaultParamspace
-	DefaultSendEnabled       = types.DefaultSendEnabled
 )
 
 var (
 	// functions aliases
-	RegisterCodec          = types.RegisterCodec
-	ErrNoInputs            = types.ErrNoInputs
-	ErrNoOutputs           = types.ErrNoOutputs
-	ErrInputOutputMismatch = types.ErrInputOutputMismatch
-	ErrSendDisabled        = types.ErrSendDisabled
-	NewMsgSend             = types.NewMsgSend
-	NewMsgMultiSend        = types.NewMsgMultiSend
-	NewInput               = types.NewInput
-	NewOutput              = types.NewOutput
-	ValidateInputsOutputs  = types.ValidateInputsOutputs
-	ParamKeyTable          = types.ParamKeyTable
+	RegisterCodec                   = types.RegisterCodec
```

Comment:
Just thinking aloud, if it does not make sense resolve this :)

Here we are aliasing and making accessible to the outer world some internal functions, do you think it makes sense to move those to the root types? 
But then I have no straightforward solution for the keeper

Reply:
The idea here @sabau is that by having `x/{module}/internal/` with an `x/{module}/alias.go` file is the following:

1. We keep a flattened structure as far as imports are concerned for downstream users (both developers and other components in the project).
2. We allow ourselves freedom in API restructuring internally w/o breaking outside contracts.
3. We signal to consumers (both developers and other components in the project) that we intended to keep these internal contracts private and only expose what's __absolutely__  needed.